database/migration/migrateData.ts

```diff
@@ -600,6 +600,51 @@ const migrateData = async () => {
         'RE_ASSIGN'
       );
     `)
+
+    console.log(' - Simplify how we handle "locked" reviews')
+    await DB.changeSchema(`
+      ALTER TABLE public.review_assignment DROP column IF EXISTS
+      is_locked;
+    `)
+    await DB.changeSchema(`
+      ALTER TYPE public.review_status RENAME to review_status_old;
+
+      CREATE TYPE public.review_status AS ENUM (
+        'DRAFT',
+        'SUBMITTED',
+        'CHANGES_REQUESTED',
+        'PENDING',
+        'DISCONTINUED'
+    );
+    `)
+
+    // The following commands all done in separate "changeSchema" functions, as
+    // individually they may fail based on current database state, but we still
+    // need to ensure subsequent commands are run
+    await DB.changeSchema(
+      `
+      ALTER TABLE review_status_history
+        ALTER COLUMN status TYPE public.review_status
+        USING status::text::public.review_status;
+      `,
+      { silent: true }
+    )
+
+    await DB.changeSchema(
+      `
+      UPDATE public.review_status_history
+        SET status = 'PENDING'
+        WHERE status = 'LOCKED';
+      `,
+      { silent: true }
+    )
+
+    await DB.changeSchema(`
+        DROP function review_status;
+      `)
```

Comment:
Which function is to be dropped?

Reply:
`review_status()` 🤷‍♂️ 
`-- Function to expose status name field on review table in GraphQL`
(Because the type depends on it, so it must be dropped first before we can drop the type)